packages/ember/tests/routing/router_service_test/non_application_test_test.js

```diff
@@ -47,19 +47,36 @@ moduleFor(
       assert.expect(5);
       assert.equal(this.routerService.get('currentRouteName'), null);
       assert.equal(this.routerService.get('currentURL'), null);
+      assert.equal(this.routerService.get('location'), null);
+      assert.equal(this.routerService.get('rootURL'), null);
+      assert.equal(this.routerService.get('currentRoute'), null);
+    }
+
+    ['@test RouterService properties of router can be accessed with default when router is present'](
+      assert
+    ) {
+      assert.expect(5);
+      let router = this.owner.lookup('router:main');
+      router.setupRouter();
```

Comment:
Some of these tests here are written with the idea that a router has been setup, so I appended this to a few of them.